api/HTMLInputElement.json

```diff
@@ -504,10 +504,10 @@
               "version_added": null
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤12.1"
```

Comment:
This is `height` and I found it suspect that there's no corresponding change for `width`. Filed https://github.com/mdn/browser-compat-data/issues/7165 about that.